src/python/WMCore/MicroService/Unified/MSTransferor.py

```diff
@@ -350,13 +358,30 @@ def makeTransferRequest(self, wflow):
                 self.logger.info(msg)
 
                 if self.msConfig.get('enableDataTransfer', True):
+                    # Force request-only subscription
+                    # to any data transfer going above some threshold (do not auto-approve)
+                    aboveWarningThreshold = self.msConfig.get('warningTransferThreshold') > 0. and \
+                        dataSize > self.msConfig.get('warningTransferThreshold')
+                    if aboveWarningThreshold and subscription.request_only != "y":
+                        subscription.request_only = "y"
+
                     # Then make the data subscription, for real!!!
                     success, transferId = self._subscribeData(subscription, wflow.getName(), dataIn['name'])
                     if not success:
                         break
                     if transferId:
                         transRec['transferIDs'].add(transferId)
 
+                    # Warn about data transfer subscriptions going above some treshold
+                    if aboveWarningThreshold:
+                        emailSubject = "[MS] Large data transfer in place for dataset: {dset}".format(dset=dataIn['name'])
```

Comment:
How about replacing the dataset name by the transfer id in the subject?
Something like
```
emailSubject = "[MS] Large pending data transfer under request id: {transferid}".format(transferid=transferId)
```